tests/console/check_system_info.pm

```diff
@@ -44,13 +44,26 @@ sub check_addons {
     foreach my $addon (@unique_addons) {
         my $name = get_addon_fullname($addon);
         record_info("$addon module fullname: ", $name);
-        $name = "sle-product-we"      if (($name =~ /sle-we/)      && !get_var("MEDIA_UPGRADE"));
+        $name = "sle-product-we"      if (($name =~ /sle-we/) && !get_var("MEDIA_UPGRADE") && is_sle('15+'));
         $name = "SLE-Module-DevTools" if (($name =~ /development/) && !get_var("MEDIA_UPGRADE"));
         my $out = script_output("zypper lr | grep -i $name", 200, proceed_on_failure => 1);
         die "zypper lr command output does not include $name" if ($out eq '');
     }
 }
 
+sub check_product {
+    my ($mypro) = @_;
+    $mypro //= uc get_var('SLE_PRODUCT');
+    my $product = '';
+    if ($mypro =~ /HPC/) {
+        $product = 'SLE-Product-HPC-' . get_var('VERSION');
```

Comment:
It's better to set a variable for get_var('VERSION'), since we don't expect to run this call twice or more.